### DISCUSSION TITLE
Not able to see UI

### DISCUSSION THREAD

**Discussion by Annepuvarunkumar** [learndevopsonline/learndevopsonline#1793] [Q&A]
After creating the target group and target group attachment i should be able to see the UI but I'm not able to see the UI, it is getting an error like 502 Bad Gateway.
Workstation Ip : 3.86.148.202
![Screenshot 2024-02-01 225756](https://github.com/learndevopsonline/learndevopsonline/assets/139056909/f8dbed0b-12b2-4ea6-8db9-1affde0f70ef)

**Comment by chaithanya1812** [comment]
what service you are running?
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/c208bfa1-2dc0-4084-be78-7b776ab40394)

**Reply by chaithanya1812** [reply]
HI Anne your frontend-service is not running ,so your DNS record failing
![WhatsApp Image 2024-02-02 at 12 55 42_c4ee22ee](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/7f95d981-7021-498e-a873-e9fa72d670ca)

**Reply by Annepuvarunkumar** [reply]
yes, but now its running, I'm able to see the UI, i have changed the port no's from 8080 to 80 of nginx configuration

**Reply by chaithanya1812** [reply]
Now it's working check-once !
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/1e05f892-f472-492a-807c-9a07ea55ceef)

**Reply by Annepuvarunkumar** [reply]
yes its working
